### PR TITLE
update the docs of Subreddit's attributes

### PR DESCRIPTION
**Pull request by kaif-00z**
## Feature Summary and Justification

Added The Missing Attribute in Subreddit class's docs.

### CODE REVIEW

**Comment by LilSpazJoekp** [comment]
Please fill out the pull request description.

**Comment by kaif-00z** [comment]
> Please fill out the pull request description.

🤦‍♂️ done

File: asyncpraw/models/reddit/subreddit.py [hunk 1]
```diff
@@ -112,6 +112,7 @@ class Subreddit(MessageableMixin, SubredditListingMixin, FullnameMixin, RedditBa
     ``description``           Subreddit description, in Markdown.
     ``description_html``      Subreddit description, in HTML.
     ``display_name``          Name of the subreddit.
+    ``icon_img``              The url of the Subreddit's icon.
```
Comment (LilSpazJoekp) [inline asyncpraw/models/reddit/subreddit.py:115]:
```suggestion
    ``icon_img``              The URL of the subreddit icon image.
```
Let's word it like this.

**Review by LilSpazJoekp — CHANGES_REQUESTED** [review]
Minor change.

Would you add this to `asyncpraw/models/reddit/user_subreddit.py` as well.

**Comment by kaif-00z** [comment]
> Minor change.
> 
> Would you add this to `asyncpraw/models/reddit/user_subreddit.py` as well.

done

**Review by LilSpazJoekp — APPROVED** [review]

**Comment by LilSpazJoekp** [comment]
Thank you!

**Pull request merged**